third_party/blink/renderer/core/frame/local_frame_mojo_handler.h

```diff
@@ -243,6 +249,9 @@ class LocalFrameMojoHandler
   // DevicePostureServiceClient implementation:
   void OnPostureChanged(device::mojom::blink::DevicePostureType posture) final;
 
+  // AuthTokenStoreObserver implementation:
+  void OnTokenChanged(const WTF::String& token) final;
```

Comment:
Nit: I'd rename these `OnRecordReplayAuthTokenChanged` - our convention has been to keep it pretty verbose and reference our own stuff.  This makes it easier to grep against in the codebase

Reply:
did a couple more passes and I think got everything to mention `RecordReplay`.